src/main/java/Base/TestBase.java

```diff
@@ -27,7 +28,7 @@ public void initialize() throws IOException {
                 System.setProperty("webdriver.chrome.driver", "C:\\IeDriverServer.exe");
                 driver = new InternetExplorerDriver();
             } else if (CONFIG.getProperty("browser").equals("chrome")) {
-                System.setProperty("webdriver.chrome.driver", "C:\\chromedriver.exe");
+                System.setProperty("webdriver.chrome.driver", "/Users/levihabila/IdeaProjects/chromedriver");
```

Comment:
This change you made to your chrome.exe path is making your branch to fail. You can make the change to be able to run to your tests locally but dont push it to the repo because it will make the whole tests to fail. At the moment your tests are failing.